mdal/CMakeLists.txt

```diff
@@ -37,6 +38,9 @@ SET(MDAL_HEADERS
   frmts/mdal_selafin.hpp
   frmts/mdal_esri_tin.hpp
   frmts/mdal_ply.hpp
+  3rdparty/libplyxx.h
+  3rdparty/libplyxx/libplyxx_internal.h
+  3rdparty/textio.h
```

Comment:
can we introduce CMake flag "WITH_LIBPLYXX", so packagers can actually decided to build MDAL without PLY (in case of any issues with license, duplicated symbols used in libplyxx library or anything?)

Reply:
I made the flag "BUILD_PLY"